tests/e2e/support/commands.js

```diff
@@ -342,6 +342,21 @@ Cypress.Commands.add('playPointsSpectator', (card, pNum) => {
     });
 });
 
+Cypress.Commands.add('playPointsById', (cardId) => {
+  return io.socket.get(
+    '/game/points',
+    {
+      cardId,
+    },
+    function handleResponse(_, jwres) {
+      if (jwres.statusCode !== 200) {
+        throw new Error(jwres.body.message);
+      }
+      return jwres;
+    },
+  );
+});
```

Comment:
The need for this command is awkward and warrants further thought. Most of our moves input a Card object like 
```
{ suit: 1, rank: 12 }
```

and then look up the `id` of the card (needed for the api call) by looking in the `game` store module clientside for a card with the same suit and rank. This doesn't work for making moves in games where the game object isn't in the store, however. So my solution was to alias the card id that I want to play _while the visible user is spectating_ and the card's id _is_ in the store, then to use this new command after we leave the game (and join a new one) to make the move in the game we aren't watching using that stored card id. 

It works, but it smells and it feels extra weird that we have some commands for doing things as a spectator where you specify which pNum and then this where you give the cardId directly. Worth investigating a cleaner way to consolidate.

@seriouslysean penny for your thoughts